hazelcast-integration/hibernate-2ndlevel-cache/pom.xml

```diff
@@ -18,9 +18,9 @@
         <!-- needed for checkstyle/findbugs -->
         <main.basedir>${project.parent.parent.basedir}</main.basedir>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <hibernate.version>5.4.10.Final</hibernate.version>
-        <hazelcast.hibernate.version>2.0.0</hazelcast.hibernate.version>
-        <h2db.version>1.4.194</h2db.version>
+        <hibernate.version>5.6.12.Final</hibernate.version>
+        <hazelcast.hibernate.version>2.2.2-SNAPSHOT</hazelcast.hibernate.version>
```

Comment:
I don't really like the - SNAPSHOT ending, I guess it's a leftover after working with snapshot version before the release?

Reply:
In other code samples we use SNAPSHOT versions so I've followed this approach, we encourage users to test new features within code samples ;)